projects/fsastorefrontstyles/scss/base/forms.scss

```diff
@@ -12,14 +12,3 @@
   font-weight: var(--cx-font-weight-semi);
   text-transform: uppercase;
 }
-
-.custom-file-input {
```

Comment:
Why move general forms styling to dynamicforms and scope it with cx-dynamic-form component ?

Reply:
Because turns out they are not general, but used only in our upload component.